src/locales/en.json

```diff
@@ -364,6 +364,7 @@
         "AddNote": "Add note",
         "AddToQueueSuccessful": "File {filename} added to Queue.",
         "AllJobs": "All",
+        "Amount": "Amount",
```

Comment:
I think `amount` is a bit inaccurate as the name. The amount of what? Could also be amount of filament used. I would suggest something like `job count`, just `jobs` or something similar. To make clear its the numbers of jobs.

Reply:
thx for the tip. I could not find of a better name.